src/applications/personalization/profile-2/components/MilitaryInformation.jsx

```diff
@@ -164,6 +165,18 @@ const MilitaryInformation = ({ militaryInformation }) => (
   </>
 );
 
+MilitaryInformation.propTypes = {
```

Comment:
@erikphansen When I was adding PropTypes to this component, it appeared that `error` is not a property on `serviceHistory`. Am I missing something or does this mean that there is a bunch of unreachable code in this component?

Reply:
I believe that gets added only if there is an error fetching the military info data